ckanext/dia_theme/templates/user/snippets/api_token_list.html

```diff
@@ -0,0 +1,15 @@
+{% ckan_extends %}
+
+{% block token_cell_actions %}
+<td>
+  {% set action = h.url_for("user.api_token_revoke", id=user['name'], jti=token['id']) %}
+  <form action="{{ action }}" method="POST">
+{{ h.csrf_input() }}
+    <div class="btn-group">
+      <button type="submit" href="{{ action }}" class="btn btn-danger btn-sm" title="{{ _('Revoke') }}" data-module="confirm-action" data-module-with-data=true>
```

Comment:
I had to override this to fix a bug where the csrf extension we are using didn't work. Adding the `data-module-with-data=true` apparently adds the injected csrf token in the ajax request to revoke the token.